src/initial_read_module.f90

```diff
@@ -2690,6 +2691,9 @@ subroutine write_info(titles, mu, vary_mu, HNL_fac, NODES)
        call chrcap(ensemblestr,3)
        write(io_lun, fmt='(4x,a15,a3," MD run for ",i5," steps ")') job_str, ensemblestr, MDn_steps
        write(io_lun, fmt='(6x,"Initial ion temperature: ",f9.3,"K")') temp_ion
+       if (temp_ion_end .ne. temp_ion) then
+         write(io_lun, fmt='(6x,"Final ion temperature: ",f9.3,"K")') temp_ion_end 
```

Comment:
This needs to be only on `ionode` (or `if(myid==0)` and only when `iprint` is high enough